client/src/components/SideBar.jsx

```diff
@@ -0,0 +1,23 @@
+import { useNavigate } from  "react-router";
+import { Tab } from "../utils/utils";
+import "../styles/SideBar.css"
+
+const SideBar = ({ handleTabSelect }) => {
+
+    const navigate = useNavigate();
+
+    return <nav className="sidebar"> 
+        <div onClick={() => {
+            handleTabSelect(Tab.EVENTS);
+        }}>
+            <h3 className="caprasimo-regular">Events</h3>
+        </div>
+        <div onClick={() => {
+            handleTabSelect(Tab.GROUPS);
+        }}>
+            <h3 className="caprasimo-regular">Groups</h3>
+        </div>
```

Comment:
in order to reduce duplicated code, you can map each `Tab` value to the component.
```suggestion
        {Object.keys(Tab).map(currentTab => (
          <div onClick={() => {
              handleTabSelect(currentTab);
          }}>
              <h3 className="caprasimo-regular">{currentTab}</h3>
          </div>
        )}

```